pkgs/development/tools/squawk/default.nix

```
@@ -0,0 +1,55 @@
+{ darwin
+, fetchFromGitHub
+, lib
+, libiconv
+, libpg_query13
+, openssl
+, pkg-config
+, rustPlatform
+, stdenv
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "squawk";
+  version = "0.20.0";
+
+  src = fetchFromGitHub {
+    owner = "sbdchd";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-v9F+HfscX4dIExIP1YvxOldZPPtmxh8lO3SREu6M+C0=";
+  };
+
+  cargoHash = "sha256-kSaQxqom8LSCOQBoIZ1iv+q2+Ih8l61L97xXv5c4a0k=";
+
+  cargoPatches = [
+    ./correct-Cargo.lock.patch
+  ];
+
+  patches = [
+    ./fix-postgresql-version-in-snapshot-test.patch
+  ];
+
+  nativeBuildInputs = [
+    pkg-config
+    rustPlatform.bindgenHook
+  ];
+
+  buildInputs = [
```

I think openssl is not needed on Darwin, because they use `Security` instead
```suggestion
  buildInputs = lib.optionals (!stdenv.isDarwin) [
```